src/components/JiraCard/components/ActivityStream.tsx

```diff
@@ -88,11 +88,11 @@ const options = {
   }
 };
 
-export const ActivityStream = () => {
+export const ActivityStream = ({projectKey}: {projectKey: string}) => {
```

Comment:
stupid question as I learn TypeScript better: why wouldn't this line just be...

```
export const ActivityStream = (projectKey: string) => {
```

...as I saw in some of the other component exports?

Reply:
You probably saw it on hooks in this plugin. React Components takes in fact object as props (function arguments), so you need to type this like other objects in TypeScript.

Reply:
thanks